### DISCUSSION TITLE
Plan to support redis 7.x ?

### DISCUSSION THREAD

**Discussion by ankitsinghal22** [Grokzen/docker-redis-cluster#143] [Q&A] [answered]
We have a requirement to use the new features that have been provided in Redis 7.x. Are there any plans to support that in near future?
We are using the cluster mode of redis, and this image is our ready to go image for this purpose.

**Comment by Grokzen** [comment]
@ankitsinghal22 If you check the README.md file there are instructions on how to build any other redis version you want. It is not guaranteed that they are supported or works 100% but it is worth a try until I add it in fully in the master branch.

```bash
# Example plain docker
docker build --build-arg redis_version=7.0.0 -t grokzen/redis-cluster .
```

should work out fine unless they have changed out something more critical deep down in how you run it.

Also the fastest way to get support into this repo is to do the work and submit a PR :)

**Reply by Grokzen** [reply, marked as answer]
Support for redis 7 images have been out for some time now, marking as resolved
